src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -235,26 +235,29 @@ object EventMessages {
 
     override def context(event: UploadEvent): IO[Unit] = IO.unit
 
-    override def govUkNotifyEmail(uploadEvent: UploadEvent, context: Unit): List[GovUKEmailDetails] = List(
-      GovUKEmailDetails(
-        templateId = govUKNotifTemplateId(uploadEvent),
-        userEmail = uploadEvent.userEmail,
-        personalisation = Map(
-          "userEmail" -> uploadEvent.userEmail,
-          "userId" -> uploadEvent.userId,
-          "transferringBodyName" -> uploadEvent.transferringBodyName,
-          "consignmentId" -> uploadEvent.consignmentId,
-          "consignmentReference" -> uploadEvent.consignmentReference,
-          "status" -> uploadEvent.status
-        ),
-        reference = s"${uploadEvent.consignmentReference}-${uploadEvent.userId}"
-      )
-    )
+    override def govUkNotifyEmail(uploadEvent: UploadEvent, context: Unit): List[GovUKEmailDetails] =
+      if (!uploadEvent.transferringBodyName.toUpperCase.contains("MOCK")) {
```

Comment:
I would consolidate this test into a function, as its being used in multiple places.

Reply:
Have added `isMockEvent` method to the UploadEventDecoder